tools/filetop.py

```diff
@@ -4,7 +4,8 @@
 # filetop  file reads and writes by process.
 #          For Linux, uses BCC, eBPF.
 #
-# USAGE: filetop.py [-h] [-C] [-r MAXROWS] [interval] [count]
+# USAGE: filetop.py [-h] [-a] [-C] [-r MAXROWS] [-p PID] [--reverse]
+#                   [--read-only] [--write-only] [interval] [count]
```

Comment:
I can see write-only and read-only are useful since sorting can be done based on either read or write but not both.
But I don't like --reverse. Currently, the sorting is in descending order which is the same for other tools e.g.  biotop.py.

Reply:
Removed --reverse